tests/installation/welcome.pm

```diff
@@ -115,6 +117,7 @@ sub run {
         if ((match_has_tag 'inst-betawarning') || (match_has_tag 'inst-welcome') || (match_has_tag 'inst-welcome-no-product-list')) {
             last;
         }
+        handle_untrusted_gpg_key;
```

Comment:
if this is not conditional, wouldn't trigger the [else](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/04be90dafbe47756869d5af8c258e6a2812649a8/lib/utils.pm#L204) path when not pop-up is displayed?

Reply:
and this sounds right in the self_update scenario. In any other case it wont print anything as the `import-known-untrusted-gpg-key` is not used. But let me run some VRs to make sure

Reply:
check http://aquarius.suse.cz/tests/3168#. Can i resolve this?

Reply:
seems fine, you're right, was previous behavior. thanks for checking.